src/components/PageReady/index.js

```diff
@@ -76,16 +70,10 @@ class PageReady extends React.Component<PropShape> {
         fetching: true,
       });
 
-      const requestParams = [
-        this.props.url,
-        this.props.method === 'post' ? { ...this.props.postBody } : {},
-      ];
-
-      axios[this.props.method.toLowerCase()](...requestParams)
-        .then((res: Object): any => {
-          const { data } = res;
+      this.props.requestInstance
```

Comment:
the problem here is that the promise is executed immediately not waiting for didMount... see previous comment for solution